zetaclient/evm/inbounds.go

```diff
@@ -59,221 +59,157 @@ func (ob *ChainClient) ObserveTrackerSuggestions() error {
 		return err
 	}
```

Comment:
I think we can simplify function name
`ObserveTrackerSuggestions` -> `ObserverTrackers` and add comment

Reply:
https://github.com/zeta-chain/node/pull/1836/commits/be3e8f9862d885050fb7ac19a3845e62bc8b27cd